django/db/models/sql/query.py

```diff
@@ -2779,3 +2779,33 @@ def update_join_types(self, query):
         query.promote_joins(to_promote)
         query.demote_joins(to_demote)
         return to_demote
+
+
+# Resolve OuterRef expressions in order_by clause by cloning and resolving them.
+def resolve_order_by_expressions(self, order_by_expressions):
+    resolved_expressions = []
+    for expr in order_by_expressions:
+        if (
+            hasattr(expr, "contains_outer_reference")
+            and expr.contains_outer_reference()
+        ):
+            clone = (
+                expr.clone()
+            )  # Clone the expression to avoid modifying the original.
```

Comment:
Would you mind updating these to take up less space? In their current form they look like black formatted an inline comment, but every (?) comment I've seen in the django repo is on its own line.
```suggestion
            # Clone the expression to avoid modifying the original.
            clone = expr.clone()
```